examples/communication/child_to_parent/src/main.rs

```diff
@@ -0,0 +1,86 @@
+use yew::{html, AttrValue, Callback, Component, Context, Html, Properties};
+
+// The `Parent` component is the parent of the `Child` component.
+
+pub struct Parent {
+    value: u32,
+}
+
+pub enum Msg {
+    ButtonClick,
+}
+
+impl Component for Parent {
+    type Message = Msg;
+    type Properties = ();
+
+    fn create(_ctx: &Context<Self>) -> Self {
+        Self { value: 0 }
+    }
+
+    fn update(&mut self, _ctx: &Context<Self>, msg: Self::Message) -> bool {
+        match msg {
+            Msg::ButtonClick => {
+                self.value += 1;
+                true
+            }
+        }
+    }
+
+    fn view(&self, ctx: &Context<Self>) -> Html {
+        let msg = format!("My children have been clicked {} times", self.value);
+        let on_clicked = ctx.link().callback(|_| Msg::ButtonClick);
+        html! {
+            <div class="app">
+                <div class="parent">
+                    <h2>{ "Child-to-Parent communication example" }</h2>
+                    <div>{msg}</div>
+                    <Child name="Alice" on_clicked={on_clicked.clone()} />
+                    <Child name="Bob" {on_clicked} />
+                </div>
+            </div>
+        }
+    }
+}
+
+// The `Child` component is the child of the `Parent` component, and will send updates to the parent using a Callback.
+
+pub struct Child;
```

Comment:
Refactor to a doc comment.

```suggestion
/// The `Child` component is the child of the `Parent` component, and will send updates to the parent using a Callback.
pub struct Child;
```

Reply:
Done